src/renderer/services/binance/service.ts

```diff
@@ -280,9 +287,18 @@ const loadAssetTxs$ = ({
   const endTime = Date.now()
   // 90 day window - similar to ASGARDEX wallet approach,
   // see https://gitlab.com/thorchain/asgard-wallet/-/blob/develop/imports/api/wallet.js#L39-48
-  const diffTime = 90 * 24 * 60 * 60 * 1000
+  const diffTime = 90 * 24 * 60 * 60 * 1001
```

Comment:
Why `1001`?

Reply:
dont know :sweat_smile:
seems some local tests which i missed to revert. few minutes pls